### PR TITLE
docs(weekly): add this week in databend 109

### PR DESCRIPTION
**Pull request by PsiACE**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

- Closes #issue

- - -
This change is [Reviewable](https://reviewable.io/reviews/datafuselabs/databend/12710)

### CODE REVIEW

File: website/blog/2023-09-03-databend-weekly-109.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+- [Docs | Understanding Cluster Key](https://databend.rs/doc/sql-commands/ddl/clusterkey/)
+- [Docs | Databend Data Storage: Snapshot, Segment, and Block](https://databend.rs/doc/sql-commands/ddl/clusterkey/)
+
+## Code Corner
+
+Discover some fascinating code snippets or projects that showcase our work or learning journey.
+
+### Exploring Databend Local Mode
+
+**Databend Local Mode** aims to provide a simple Databend environment where users can directly interact with Databend using SQL without the need to deploy Databend services. This makes it convenient for developers to perform basic data processing using Databend.
```
Comment (soyeric128) [inline website/blog/2023-09-03-databend-weekly-109.md:64]:
```suggestion
**Databend Local Mode** aims to provide a simple Databend environment where users can directly interact with Databend using SQL without the need to deploy Databend services. This makes it convenient for developers to perform basic data processing using Databend.
```
```suggestion
**Databend Local Mode** aims to provide a simple Databend environment where users can directly interact with Databend using SQL without the need to perform a full Databend deployment. This makes it convenient for developers to perform basic data processing using Databend.
```

File: website/blog/2023-09-03-databend-weekly-109.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+In local mode, a temporary `databend-query` process is started, which integrates both the client and server support. The storage is located in a temporary directory and its lifecycle follows the process. Once the process ends, the resources are also destroyed. You can start multiple local processes on one server, and their resources are isolated from each other.
```
Comment (soyeric128) [inline website/blog/2023-09-03-databend-weekly-109.md:66]: databend-local starts a temporary `databend-query` process which functions as both meta and query services.

File: website/blog/2023-09-03-databend-weekly-109.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+│ UInt8 │ UInt8 │    UInt64 NULL    │
+├───────┼───────┼───────────────────┤
+│     0 │     0 │ 41666666833333332 │
+│     1 │     0 │ 41666666166666668 │
+│     2 │     0 │ 41666666500000000 │
+└───────────────────────────────────┘
+0 row result in 1.669 sec. Processed 1 billion rows, 953.67 MiB (599.02 million rows/s, 4.46 GiB/s)
+```
+
+If you need to use Databend in a production environment, we recommend deploying the Databend service according to the official documentation or using Databend Cloud. However, if you are a developer or a testing engineer, you can experience Databend using local mode.
```
Comment (soyeric128) [inline website/blog/2023-09-03-databend-weekly-109.md:88]:
```suggestion
If you need to use Databend in a production environment, we recommend deploying a Databend cluster according to the official documentation or using Databend Cloud. databend-local is a good choice for developers to quickly leverage Databend's capabilities without the need to deploy a full Databend instance.
```

File: website/blog/2023-09-03-databend-weekly-109.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+- [PR | chore(query): improve databend local](https://github.com/datafuselabs/databend/pull/12659)
+
+## Highlights
+
+We have also made these improvements to Databend that we hope you will find helpful:
+
+- Preliminary support for `MERGE INTO`.
+- Implemented SQLsmith testing framework to support more accurate fuzzy testing.
+- Read the document [Docs | Setting Environment Variables](https://databend.rs/doc/deploy/node-config/environment-variables) to learn how to change Databend configuration through environment variables.
```
Comment (soyeric128) [inline website/blog/2023-09-03-databend-weekly-109.md:100]:
```suggestion
- Read the document [Docs | Setting Environment Variables](https://databend.rs/doc/deploy/node-config/environment-variables) for how to change Databend configurations through environment variables.
```

File: website/blog/2023-09-03-databend-weekly-109.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+- Implemented `json_strip_nulls` and `json_typeof` functions. You can also read [Docs | Semi-Structured Functions](https://databend.rs/doc/reference/functions/variant-functions) to understand the functions designed and implemented by Databend for semi-structured data processing.
```
Comment (soyeric128) [inline website/blog/2023-09-03-databend-weekly-109.md:101]:
```suggestion
- Implemented `json_strip_nulls` and `json_typeof` functions. You can also read [Docs | Semi-Structured Functions](https://databend.rs/doc/reference/functions/variant-functions) to check out Databend functions for semi-structured data processing.
```

File: website/blog/2023-09-03-databend-weekly-109.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+
+If you are interested in learning more, please check out the resources listed below:
+
+- [PR | chore(query): improve databend local](https://github.com/datafuselabs/databend/pull/12659)
+
+## Highlights
+
+We have also made these improvements to Databend that we hope you will find helpful:
+
+- Preliminary support for `MERGE INTO`.
```
Comment (soyeric128) [inline website/blog/2023-09-03-databend-weekly-109.md:98]: Added initial support for MERGE INTO.

File: website/blog/2023-09-03-databend-weekly-109.md (hunk 2 above)
Comment (PsiACE) [reply website/blog/2023-09-03-databend-weekly-109.md]: both client and server.

**Review by soyeric128 — APPROVED** [review]

**Pull request merged**